### ISSUE TITLE
If EF Core configures different column name for a entity property , the corresponding value of audit property is null

### ISSUE THREAD

**Issue by Li7ye** [thepirat000/Audit.NET#586]
**Describe the bug**
I have an entity : `Foo`

```
public class Foo
{
   public int Id { get; set; }
    public string Content { get; set; }
    public string Name { get; set; }
}
```

audit entity : `AuditFoo`

```
public class AuditFoo
{
    [Key]
    public Guid AuditId { get; set; }

    public int Id { get; set; }
    public string Content { get; set; }
    public string Name { get; set; }
}
```

In my business dbcontext , I want to use "CONTENT" name as database column name.
```
protected override void OnModelCreating(ModelBuilder modelBuilder)
    {
        modelBuilder.Entity<Foo>(e =>
        {
            e.Property(x => x.Content)
             .HasColumnName("CONTENT");
        });
    }
```

I try to use explicit map to setup `AuditFoo` , but I always get null value for `AuditFoo.Content`.
```
x.AuditTypeExplicitMapper(c =>
{
    c.MapExplicit<AuditFoo>(entry => entry.EntityType == typeof(Foo), (entry, auditFoo) =>
     {
              //  auditFoo.Content is null ?
             //   auditFoo.Name has value 
     });
}
```

**To Reproduce**

refer to attachment.  

**Expected behavior**

audit property should has same value on entity level, despite the column name is different.  

**Libraries (specify the Audit.NET extensions being used including version):**
 - Audit.EntityFramework.Core: 20.2.2

**Target .NET framework:**
 - .NET 6

[DontMatchPropertyWhenHasColumnName.zip](https://github.com/thepirat000/Audit.NET/files/10988990/DontMatchPropertyWhenHasColumnName.zip)

**Comment by thepirat000** [comment]
This was fixed in version 20.2.3, please upgrade your references and re-test

**Comment by Li7ye** [comment]
It's working now. Thanks.
